scripts/benchall.py

```
@@ -0,0 +1,132 @@
+        bench.run_applicable(dataset_name)
+        #bench.run_everything(['ingest', 'cold'])
+    except Exception as e:
+        statement = f"{datetime.datetime.now().strftime('%Y-%m-%d %H:%M:%S')}: {bencher.__name__} with argument {kwargs} failed on dataset {dataset_name}: {type(e).__name__}: {str(e)}"
+        with open((current_dir / 'exceptions.log').resolve(), 'a') as file:
+            file.write(f"{statement}\n")
+        print(statement)
+        if attach:
+            if bench is not None:
+                bench.docker_attach()
+        else:
+            raise e
```

Ruff (0.12.2)

Use `raise` without specifying exception name

Remove exception name

(TRY201)